route.go

```diff
@@ -110,6 +110,23 @@ func (r *RouteService) Delete(req *RouteMessage) error {
 	return nil
 }
 
+// Get Route(s)
+func (r *RouteService) Get(req *RouteMessage) ([]RouteMessage, error) {
+	flags := netlink.Request
```

Comment:
Shouldn't this be `flags := netlink.Request | netlink.DumpFiltered` ?

Reply:
In testing, the flag seemed to not change the behavior.  I can add it if its technically 'correct'.